client/src/containers/Jamroom.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+class Jamroom extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+
+  render() {
+    return (
```

Comment:
Please take the time to appropriately indent any nested elements. Two spaces per level is a good convention.